frontend/src/providers/AuthProvider.vue

```diff
@@ -84,29 +87,26 @@ const logout = async () => {
     authState.value = {
         isAuthenticated: false,
         famLoginUser: null,
-        cognitoUser: null,
-        accessToken: null,
-        idToken: null,
-        refreshToken: null,
```

Comment:
RefreshToken no longer exists anymore in aws-amplify v6.  For the cognitoUser, accessToken and idToken, I didn't see we use them anywhere, so removed them